website/docs/docs/build/data-tests.md

```diff
@@ -251,6 +261,29 @@ Note that, if you select to store test failures:
 * Test result tables are created in a schema suffixed or named `dbt_test__audit`, by default. It is possible to change this value by setting a `schema` config. (For more details on schema naming, see [using custom schemas](/docs/build/custom-schemas).)
 - A test's results will always **replace** previous failures for the same test.
 
+<VersionBlock firstVersion="1.8" lastVersion="1.8">
+
+## New `tests:` syntax
+
+Data tests were historically called "tests" in dbt as the only form of testing available. With the introduction of unit tests in v1.8, it was necessary to update our naming conventions and syntax. As of v1.8,  `tests:` is still supported in your YML configuration file as an alias but will be deprecated in the future in favor of `data_tests:`. 
+
+As we progress towards this deprecation, the examples in our docs pages will be updated to reflect this new syntax, but we highly recommend you begin the migration process as soon as you upgrade to v1.8 to avoid interruptions or issues in the future.
+
+```yml
+
+models:
+
+name: orders
```

Comment:
the yml spacing looks a bit weird here

Reply:
![Screenshot 2024-02-29 at 7 08 53 PM](https://github.com/dbt-labs/docs.getdbt.com/assets/53586774/01db98d1-97cf-4b9b-a2fe-9cf2d6df03bf)